nexus/tests/unit/test_gencell.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+import numpy as np
+from forlib.gencell import nsupercell, generate_all_tilematrices
+from forlib.gencell import generate_all_supercells
+
```

Comment:
Move contents to existing "structure" function in nxs-test.  Replace assert with nassert().  Check that "nxs-test -R structure" passes.